fruit_classifier/predict/predict_utils.py

```diff
@@ -96,3 +98,33 @@ def load_classifier():
     model = load_model(str(model_path))
 
     return model
+
+
+def pick_random_image(from_folder):
+    """
+    Picks a random image from a random directory in "from_folder"
+
+    Note, this includes images from both training set and test set.
+
+    Parameters
+    ----------
+    from_folder : str - the folder where an image is found
+
+    Returns
+    -------
+    image_path : str - the path to the image
+
+    """
+    generated_data_dir = \
+        Path(__file__).absolute().parents[2].joinpath(
+            'generated_data')
+    cleaned_dir = generated_data_dir.joinpath(from_folder)
```

Comment:
Is there a reason why this is called `cleaned_dir`? It could be misleading when reading the code when the input is unknown

Reply:
Changed to "image_dir".

Reply:
Also, slightly simplified code of that function.